kubernetes/table_kubernetes_custom_resource.go

```diff
@@ -0,0 +1,132 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableKubernetesCustomResource(ctx context.Context) *plugin.Table {
+	crdName := ctx.Value(contextKey("CRDName")).(string)
```

Comment:
It doesn't look like it, but to confirm, we're no longer adding any prefix for CRD table names right?

Reply:
yes